src/super_gradients/common/sg_loggers/deci_platform_sg_logger.py

```diff
@@ -28,6 +28,7 @@ def __init__(
         training_params: dict,
         checkpoints_dir_path: str,
         model_name: str,
+        upload_model: bool = True,
```

Comment:
not really related to this specific change. but can you please add parameters documentation

Reply:
I added doc for this logger, and at the same time updated other loggers which were missing some fields: https://github.com/Deci-AI/super-gradients/pull/799/commits/386678f10d0d2d85e0065ee4b7c095f8a86699be